### PR TITLE
[mrg] [ref#113] added doc dependencies 

### PR DESCRIPTION
**Pull request by the872**
as per [#113](https://github.com/pavanramkumar/pyglmnet/issues/113)

### CODE REVIEW

File: doc/developers.rst [hunk 1]
```diff
@@ -18,13 +18,23 @@ with more coverage.
 
 Building Documentation
 ----------------------
+The following is needed in order to build the documentation.
+
+*    ``pip install sphinx``
+*    ``pip install sphinx-gallery``
+*    ``pip install Image``
```
Comment (jasmainak) [inline doc/developers.rst]: why is this a dependency @the872 ? which part of the code requires this?

Comment (the872) [inline doc/developers.rst]: @jasmainak sphinx for building docs, sphinx-gallery for compiling the .py examples, Image provides the graph thumbnails

Comment (jasmainak) [inline doc/developers.rst]: reading through the `sphinx-gallery` docs, it seems you need `pillow`: https://github.com/sphinx-gallery/sphinx-gallery not `Image`.

File: doc/developers.rst [hunk 2]
```diff
@@ -18,13 +18,23 @@ with more coverage.
 
 Building Documentation
 ----------------------
+The following is needed in order to build the documentation.
+
+*    ``pip install sphinx``
```
Comment (jasmainak) [inline doc/developers.rst]: you can put all the installers in the same line, no? something like:
```sh
pip install sphinx sphinx-gallery pillow
```
does that work?

**Pull request merged**